uiTestHelpers/mobileStepDefinitions/actions/setElementInsideElement.js

```diff
@@ -1,6 +1,6 @@
-// module.exports = function setElementInsideElement(locatorKey2, locatorKey, value) {
-//   return this.getCurrentPage()
-//     .getElementWhenInDOM(locatorKey, locatorKey2)
-//     .then((element) => element.clear()
-//       .then(() => element.sendKeys(value)));
-// };
+module.exports = async function setElementInsideElement(locatorKey2, locatorKey, value) {
+  const pageObj = await this.getCurrentPage();
```

Comment:
For consistency, shouldn't the setElementInsideElement function take the arguments in the same order as getElementInsideElement? Like key1, key2

Reply:
see other comment

Reply:
see other comment